tekton/predicates_test.go

```diff
@@ -114,7 +114,7 @@ var _ = Describe("Predicates", func() {
 			Expect(instance.Update(contextEvent)).To(BeFalse())
 		})
 
-		It("should return false when an updated event is received for a succeeded PipelineRun and not signed", func() {
+		It("should return true when an updated event is received for a succeeded PipelineRun and not signed", func() {
```

Comment:
can we rephrase it to something like `when an updated event is received for a succeeded PipelineRun with failed chains signing` ?

Reply:
Done.